pkgs/development/python-modules/filemagic/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub, file
+, mock }:
+
+buildPythonPackage rec {
+  pname = "filemagic";
+  version = "1.6";
+
+  # Don't use the PyPI source because it's missing files required for testing
+  src = fetchFromGitHub {
+    owner = "aliles";
+    repo = "filemagic";
+    rev = "138649062f769fb10c256e454a3e94ecfbf3017b";
+    sha256 = "1jxf928jjl2v6zv8kdnfqvywdwql1zqkm1v5xn1d5w0qjcg38d4n";
+  };
+
+  postPatch = ''
+    substituteInPlace magic/api.py --replace "ctypes.util.find_library('magic')" \
+      "'${file}/lib/libmagic${stdenv.hostPlatform.extensions.sharedLibrary}'"
+  '';
+
+  checkInputs = [ mock ];
```

Comment:
We need to constrain `unittest2` like https://github.com/aliles/filemagic/blob/138649062f769fb10c256e454a3e94ecfbf3017b/setup.py#L54

Reply:
Fixed: https://github.com/NixOS/nixpkgs/commit/78af4fb103a1e3748ef86d6f8c174ec11cb64f9f